Failed to execute `jbang info tools/classpath` for JShell script

demo.jsh code as following: 

```java
///usr/bin/env jbang "$0" "$@" ; exit $?

//DEPS org.apache.commons:commons-lang3:3.12.0

import org.apache.commons.lang3.StringUtils;

    System.out.println(StringUtils.center("Hello World", 20));

```

You can execute demo.jsh by `jbang demo.js`.    But failed to execute `jbang info tools --verbose demo.jsh` to get ScriptInfo from JShell script, and stack trace as following: 

```
[jbang] jbang version 0.85.1
[jbang] [ERROR] public class java.lang.NullPointerException
java.lang.NullPointerException
	at dev.jbang.cli.BaseInfoCommand$ScriptInfo.<init>(Info.java:113)
	at dev.jbang.cli.BaseInfoCommand.getInfo(Info.java:154)
	at dev.jbang.cli.Tools.doCall(Info.java:168)
	at dev.jbang.cli.BaseCommand.call(BaseCommand.java:93)
	at dev.jbang.cli.BaseCommand.call(BaseCommand.java:15)
	at picocli.CommandLine.executeUserObject(CommandLine.java:1953)
	at picocli.CommandLine.access$1300(CommandLine.java:145)
	at picocli.CommandLine$RunLast.executeUserObjectOfLastSubcommandWithSameParent(CommandLine.java:2352)
	at picocli.CommandLine$RunLast.handle(CommandLine.java:2346)
	at dev.jbang.cli.JBang$3.handle(JBang.java:145)
	at dev.jbang.cli.JBang$3.handle(JBang.java:140)
	at picocli.CommandLine$AbstractParseResultHandler.execute(CommandLine.java:2179)
	at picocli.CommandLine.execute(CommandLine.java:2078)
	at dev.jbang.Main.main(Main.java:14)
[jbang] If you believe this a bug in jbang, open an issue at https://github.com/jbangdev/jbang/issues
```

Related issue:  https://github.com/jbangdev/jbang-idea/issues/10

--- a/src/main/java/dev/jbang/cli/Info.java
+++ b/src/main/java/dev/jbang/cli/Info.java
@@ -110,7 +110,7 @@ abstract class BaseInfoCommand extends BaseScriptCommand {
 				description = ss.getDescription().orElse(null);
 
 				if (ctx != null) {
-					applicationJar = src.getJarFile().getAbsolutePath();
+					applicationJar = src.getJarFile() == null ? null : src.getJarFile().getAbsolutePath();
 					mainClass = ctx.getMainClassOr(src);
 					requestedJavaVersion = src.getJavaVersion();
 					availableJdkPath = Objects.toString(JdkManager.getCurrentJdk(requestedJavaVersion), null);

--- a/src/test/java/dev/jbang/cli/TestInfo.java
+++ b/src/test/java/dev/jbang/cli/TestInfo.java
@@ -126,4 +126,21 @@ public class TestInfo extends BaseTest {
 				hasSize(equalTo(1)),
 				everyItem(containsString("picocli"))));
 	}
+
+	@Test
+	void testInfoJShell() {
+		String src = examplesTestFolder.resolve("basic.jsh").toString();
+		JBang jbang = new JBang();
+		CommandLine.ParseResult pr = new CommandLine(jbang).parseArgs("info", "tools", src);
+		Tools tools = (Tools) pr.subcommand().subcommand().commandSpec().userObject();
+		BaseInfoCommand.ScriptInfo info = tools.getInfo();
+		assertThat(info.originalResource, equalTo(src));
+		assertThat(info.applicationJar, equalTo(null));
+		assertThat(info.backingResource, equalTo(src));
+		assertThat(info.javaVersion, is(nullValue()));
+		assertThat(info.mainClass, is(nullValue()));
+		assertThat(info.resolvedDependencies, Matchers.<Collection<String>>allOf(
+				hasSize(equalTo(1)),
+				everyItem(containsString("commons-lang3"))));
+	}
 }

